metrics/processors/node_aggregator.go

```diff
@@ -42,8 +42,7 @@ func (this *NodeAggregator) Process(batch *core.DataBatch) (*core.DataBatch, err
 				node, found := batch.MetricSets[nodeKey]
 				if !found {
 					glog.Warningf("Failed to find node: %s", nodeKey)
```

Comment:
When can we get into this state? Is this really a warning or lower log level?

I'd also prefer to just continue if not found rather than `if else`.